templates/user/heatmap.tmpl

```diff
@@ -1,8 +1,6 @@
-{{if .HeatmapData}}
-	<div id="user-heatmap" data-heatmap-data="{{Json .HeatmapData}}">
```

Comment:
can you revert this file back to master

or what was the reason to do this?

Reply:
It's not directly related, I just stumbled over this. From the commit message:
> always show heatmp, even if it contains no data 
> 
> rendering is wanted for freshly created users.
> rendering might be unwanted, if doer has no permission to view anyway
If you think this behaviour isn't better we can revert that

Reply:
I dont like the new behavior